bin/acfg1.py

```diff
@@ -164,12 +164,10 @@ def _apply_metadata(source, destination, hostclasses, dryrun=False):
 
 def _higher_priority(new_hostclass, old_hostclass, all_hostclasses):
     if new_hostclass:
-        if old_hostclass:
```

Comment:
I guess this is just stylistic, but wouldn't this be valid as well?

```python
if new_hostclass:
    if old_hostclass:
        return all_hostclasses.index(new_hostclass) < all_hostclasses.index(old_hostclass)
    return True
return False
```

Seems slightly more reasonable to me to continue to spread out the if-statements.